.github/workflows/check-bump-interface-version.yml

```diff
@@ -0,0 +1,25 @@
+name: check-bump-interface-version
+
+on:
+  push:
+    paths: 'stellar-contract-env-common/src/env.rs'
```

Comment:
You can probably omit this as having this check run on the main branch is probably defeating the purpose a bit and given our current setup I don't think we'll get a notification of it. We need to catch this in PRs and we can because we require PRs to be up-to-date anyway.
```suggestion
```